Base/home/anon/Documents/emoji.txt

```diff
@@ -55,6 +55,10 @@ Faces & Misc Emoji
 🦁 - U+1F981 LION FACE
 🦄 - U+1F984 UNICORN FACE
 🧋 - U+1F9CB BUBBLE TEA
+🩱 - U+1FA71 ONE-PIECE SWIMSUIT
+🩲 - U+1FA72 BRIEFS
+🩳 - U+1FA73 SHORTS
+🩹 - U+1FA79 ADHESIVE BANDAGE
 🩸 - U+1FA78 DROP OF BLOOD
```

Comment:
I think these two should switch places?

Reply:
Thanks for the heads up, we can't have that unsorted now can we? ;) Fixed now.